src/eHealth.couchQuery/services/lucene-query-factory.js

```diff
@@ -104,9 +105,12 @@ angular
                   .then(function(response) {
                     return response.data;
                   });
-              }, initialParams);
+              }, initialParams, initialOptions);
             } else {
               initialParams.q = q;
+              angular.extend({
```

Comment:
Hum ... i think that this will have no effect because the destination variable is anonymous and you are not saving the result. Anyway, why not doing this at line 92 so that defaults are clearer? Reading line 92 one could think that if `initialOptions` is not defined it gets value `{}`, while if it is not defined it will get value `{ unique: true }`